label_studio_converter/__init__.py

```diff
@@ -5,4 +5,4 @@
     print(e)
 
 
-__version__ = '0.0.45.rc5'
+__version__ = '0.0.45.rc6'
```

Comment:
Do you know how this versioning works? Are we going to change this to__version__ = `0.0.45` when everything for DEV-3578 is approved?

Reply:
That is a good question. I was referencing previous PRs and only the rc was bumped, not sure why that is but I will check!